client/js/components/procedure/SegmentsBulkEdit/SelectedTagsList.vue

```diff
@@ -10,8 +10,8 @@
 <template>
   <ul class="o-list">
     <li
-      class="inline-block border bg-color--white color--grey-dark"
-      style="border-radius: 4px; padding: 4px 8px; margin-right: 12px;"
+      class="inline-block border bg-color--white color--grey-dark rounded-md"
+      style="padding: 4px 8px; margin-right: 12px;"
```

Comment:
Could we remove the style here completely?

Reply:
sure! 5873417dae98012b0e20893dc53e67975bb1a69e